### PR TITLE
Option to display temperature in Kelvin.

### LINKED ISSUE #24815: Missing temperature option.
Game version: 7687

Operating system: windows 10 pro 64bit

Mods active: DDA

### Expected behavior
An option to switch between fahrenheit, Celsius, and kelvin 

### Actual behavior
There is no kelvin option for temperature. 

### Steps to reproduce the behavior

Go into the interface settings menu

### PR DESCRIPTION
**Pull request by OzoneH3**
#### Summary
```SUMMARY: Interface "Option to display temperature in Kelvin."```  

#### Purpose of change
Fixes #24815

#### Additional context
Will it be used? Maybe. Does it make sense? There are Cups and Quarts for volume so this just adds another fantasy number that someone can use if they desire. :P

### CODE REVIEW

**Comment by Mecares** [comment]
While you are messing with measurement units could you add the option to measure vehicle speed in tiles per turn instead of mph or kph? I would like that quite a lot.

**Comment by OzoneH3** [comment]
I've looked into the vehicle speed thing (which is all over the place and somehow uses moves combined with mph * 100 as velocity) and it seems like this gets me the right amount:
```tiles/turn = mph / 10```

If someone actually understanding the vehicle code can confirm that I'll add it.

**Comment by EhNuhc** [comment]
We are playing a physical game!

File: src/cata_utility.h [hunk 1]
```diff
@@ -227,6 +227,16 @@ double convert_volume( int volume, int *out_scale );
  */
 double temp_to_celsius( double fahrenheit );
 
+/**
+ * Convert a temperature from degrees Fahrenheit to degrees Kelvin.
+ *
+ * @param fahrenheit Temperature in degrees F.
```
Comment (BevapDin) [inline src/cata_utility.h]: Do you really this and the `@returns` line? The first comment line looks sufficient, it explains what the functions does, what it requires and what it returns.

Comment (OzoneH3) [reply src/cata_utility.h]: Thats a copy of the temp_to_celsius comment, should i remove it there as well?

Comment (BevapDin) [reply src/cata_utility.h]: Yes. Or is it necessary?

**Comment by cake-pie** [comment]
Vehicle speed units conversion may be affected in ongoing work to revamp vehicle speed calculations #25652, so suggest holding off on that one for now.

**Comment by mlangsdorf** [comment]
Vehicle speed in tiles per turn isn't changing, just how the maximum is calculated.

10 mph is 1 tile/turn. Go ahead and make the change.

**Pull request merged**